compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -477,14 +477,36 @@ class ComposeScene internal constructor(
     /**
      * Returns the current content size
      */
+    // TODO: make calculateContentSize stable in 1.7 and mark it as
+    //  @Deprecated("Use calculateContentSize() instead", replaceWith = ReplaceWith("calculateContentSize()"))
```

Comment:
It seems to me that "current content size" and "preferred size" (in infinite constraints) are not at all the same thing. If so, why do we want to deprecate getting the current size?

Reply:
Yes, there wasn't an intention to provide current content size, and I am not aware of use cases where we need it. It was added to determine the size of the window/panel before its construction.